ci/coq-tests.el

```diff
@@ -180,19 +193,30 @@ For example, COMMENT could be (*test-definition*)"
 
 (ert-deftest 020_coq-test-definition ()
   "Test *response* output after asserting a Definition."
-  (coq-fixture-on-file
-   (coq-test-full-path "test_stepwise.v")
+  (coq-fixture-on-file 
+   (coq-test-full-path "test_stepwise.v") 
    (lambda ()
      (coq-test-goto-before "(*test-definition*)")
      (proof-goto-point)
      (proof-shell-wait)
      (coq-should-response "trois is defined"))))
 
 
+(ert-deftest 021_coq-test-regression-goto-point ()
+  "Regression test for proof-goto-point after a comment, PR #90"
+  (coq-fixture-on-file 
+   (coq-test-full-path "test_stepwise.v") 
+   (lambda ()
+       (coq-test-goto-after "(*test-definition*)")
+       (proof-goto-point)
+       (proof-shell-wait)
+       t)))
+
+
 (ert-deftest 030_coq-test-position ()
   "Test locked region after Qed."
-  (coq-fixture-on-file
-   (coq-test-full-path "test_stepwise.v")
+  (coq-fixture-on-file 
```

Comment:
please remove the trailing spaces you added after each `(coq-fixture-on-file ` and `"test_stepwise.v") `, see URL https://github.com/ProofGeneral/PG/pull/490/files in order to spot them.